src/test/java/org/mule/service/http/impl/functional/server/HttpServerErrorRequestsTestCase.java

```diff
@@ -73,12 +73,14 @@ public void tearDown() {
 
   @Test
   public void methodNotAllowed() throws Exception {
-    verifyErrorResponse(new HttpPost(), "test", METHOD_NOT_ALLOWED, "Method not allowed for endpoint: /test");
+    verifyErrorResponse(new HttpPost(), "test", METHOD_NOT_ALLOWED,
+                        format("Method not allowed for endpoint: http://localhost:%s/test", port.getValue()));
```

Comment:
for generating the error message, the extra generatiod information should not be there to avoid confusion